pkgs/applications/editors/jupyter-kernels/clojupyter/default.nix

```diff
@@ -8,8 +8,11 @@
 , imagemagick
 }:
 
-# To test:
-# $(nix-build --no-out-link -E 'with import <nixpkgs> {}; jupyter.override { definitions = { clojure = clojupyter.definition; }; }')/bin/jupyter-notebook
+# Jupyter console:
+# nix run --impure --expr 'with import <nixpkgs> {}; jupyter-console.withSingleKernel clojupyter.definition'
+
+# Jupyter notebook:
+# nix run --impure --expr 'with import <nixpkgs> {}; jupyter.override { definitions = { clojure = clojupyter.definition; }; }'
```

Comment:
```suggestion
# nix run --impure --expr 'with import <nixpkgs> {}; jupyter.override { definitions.clojure = clojupyter.definition; }'
```

Reply:
I think it actually decreases the clarity slightly to use the trickier aspects of `override` like this. 

I had to double-check whether it results in the single `clojure` kernel, or the default kernel set *plus* `clojure`.

Reply:
You don't need to be an expert in nix to know this, a beginner should know this otherwise they will have some trouble looking at other peoples nixos configs.

Reply:
I hate to break it to you but looking at other people's NixOS configs isn't the most beginnery activity either :P

Incidentally, do you have some kind of tool that lints a PR and then automatically adds the suggestions on GitHub? Just curious.

I'm still waiting for your feedback on a couple other comments here, any chance you could take a look? Thanks!

Reply:
> I hate to break it to you but looking at other people's NixOS configs isn't the most beginnery activity either :P

also from documentation etc

> Incidentally, do you have some kind of tool that lints a PR and then automatically adds the suggestions on GitHub? Just curious.

no, I don't know why people keep thinking that

Reply:
> I don't know why people keep thinking that

I wondered if you were manually typing out these very detailed suggestions rather than just pointing out one instance and letting me deal with the others. I've never seen that before. Anyway, appreciate the thoroughness :)